Resources/Prototypes/_NF/Shipyard/nook.yml

```
@@ -0,0 +1,44 @@
+# Author Info
+# GitHub: Chervjakys
+# Discord: Chervjakys
+
+# Maintainer Info
+# GitHub: Chervjakys
+# Discord: Chervjakys
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: Nook
+  parent: BaseVessel
+  name: WSE Nook
+  description: A ship designed as a mobile home, with space for user modifications. Fast and comfortable.
+  price: 22000 ###base sell price is around 17000
+  category: Small
+  group: Shipyard
+  shuttlePath: /Maps/_NF/Shuttles/nook.yml
+  guidebookPage: ShipyardNook
+  class:
+  - Civilian
+  engine:
+  - Uranium
+
+- type: gameMap
+  id: Nook
+  mapName: 'WSE Nook'
+  mapPath: /Maps/_NF/Shuttles/nook.yml
+  minPlayers: 0
+  stations:
+    Nook:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Nook {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```

```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.